go.mod

```diff
@@ -77,3 +77,5 @@ require (
 	google.golang.org/grpc v1.48.0 // indirect
 	google.golang.org/protobuf v1.28.0 // indirect
 )
+
+replace github.com/grafana/grafana-api-golang-client v0.9.1 => ../grafana-api-golang-client
```

Comment:
Depends on: https://github.com/grafana/grafana-api-golang-client/pull/103

This PR is in draft until the above is submitted and a release is made.